packages/form-builder/src/blocks/fields/amount/Edit.tsx

```diff
@@ -1,51 +1,154 @@
+                        period: <RecurringPeriod count={interval} />,
+                        count: countElement,
+                        strong: <strong />, // Required to interpolate the strong tag around the "payments" text.
+                    })
+                }
+            </Notice>
+        );
+    };
+
+    const FixedRecurringMessage = () => (
+        <Notice>
+            {createInterpolateElement(__('This donation is <amount/> every <period/>. ', 'give'), {
+                amount: <strong>{amountFormatted}</strong>,
+                period: <RecurringPeriod count={parseInt(recurringBillingInterval)} />,
+            })}
+        </Notice>
+    );
+
+    const BillingPeriodControl = ({options}) => {
+        const [option, setOption] = useState(options[0]);
+        return (
+            <RadioControl
+                className={'give-billing-period-control'}
+                label={__('Billing Period', 'give')}
+                hideLabelFromVision={true}
+                selected={option}
+                options={['one-time'].concat(options).map((option) => {
+                    return {
+                        label: 'one-time' === option ? __('One Time', 'give') : periodLookup[option].adjective,
+                        value: option,
+                    };
+                })}
+                onChange={(value) => setOption(value)}
+            />
+        );
     };
 
     return (
-        <>
-            <div style={{display: 'flex', flexDirection: 'column', gap: '20px'}}>
-                {!!isFixedAmount && !customAmount && (
-                    <div style={{backgroundColor: 'var(--givewp-gray-20)', padding: '12px 16px', borderRadius: '5px'}}>
-                        <FixedPriceMessage />
-                    </div>
-                )}
-                {!!isMultiLevel && levels.length > 0 && (
-                    <LevelGrid>
-                        {levels.map((level, index) => {
-                            const levelAmount = formatCurrencyAmount(level);
-
-                            return <LevelButton key={index}>{levelAmount}</LevelButton>;
-                        })}
-                    </LevelGrid>
-                )}
-                {!!customAmount && (
-                    <div>
-                        <CurrencyControl value={setPrice} label={__('Custom amount', 'give')} hideLabelFromVision />
-                    </div>
-                )}
-            </div>
+        <div style={{display: 'flex', flexDirection: 'column', gap: '24px'}}>
+            {isRecurringDonor && <BillingPeriodControl options={recurringBillingPeriodOptions} />}
+
+            {isMultiLevel && <DonationLevels />}
+            {customAmount && <CustomAmount />}
+
+            {isMultiLevel && recurringEnabled && <RecurringMessage />}
+
+            {isFixedAmount && recurringEnabled && <FixedRecurringMessage />}
+            {isFixedAmount && !recurringEnabled && !customAmount && <FixedPriceMessage />}
```

Comment:
I think displaying the fixed messaging needs to be updated a bit ( now that preset option is removed). We only need to show a message for recurring if any of these are true:

Display Fixed Recurring Message
- recurring enabled &&
- admin defined
- or if number of payments > 0
- or if billing interval > 1

We also don't need the `You have chosen to make this donation one-time.` message since that does not translate to the front-end.